src/components/SearchMap.js

```diff
@@ -10,16 +10,16 @@ import {Button} from 'react-native-elements';
 // our components - core
 // our components - additional
 
-const SearchMap = ({mapLoaded, region, updateRegion, sites}) => {
+const SearchMap = ({mapLoaded, lastKnownRegion, updateRegion, sites}) => {
```

Comment:
Refactored across the app to rename `region` to `lastKnownRegion` since we're no longer setting `MapView`'s state from the current `region`.